test/e2e/pkg/tests/devworkspaces_tests.go

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package tests
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/devfile/api/pkg/apis/workspaces/v1alpha1"
+	"github.com/devfile/devworkspace-operator/test/e2e/pkg/config"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Create OpenShift Web Terminal Workspace]", func() {
+	defer ginkgo.GinkgoRecover()
+
+	ginkgo.It("Wait DewWorkspace Webhook Server Pod", func() {
+		controllerLabel := "app.kubernetes.io/name=devworkspace-webhook-server"
+
+		deploy, err := config.AdminK8sClient.WaitForPodRunningByLabel(config.OperatorNamespace, controllerLabel)
+		if err != nil {
+			ginkgo.Fail(fmt.Sprintf("cannot get the Pod status with label %s: %s", controllerLabel, err.Error()))
+			return
+		}
+
+		if !deploy {
+			ginkgo.Fail("Devworkspace webhook  didn't start properly")
+		}
+	})
+
+	ginkgo.It("Add OpenShift web terminal to cluster and wait running status", func() {
+		commandResult, err := config.DevK8sClient.OcApplyWorkspace(config.WorkspaceNamespace, "samples/web-terminal.yaml")
+		if err != nil {
+			ginkgo.Fail(fmt.Sprintf("Failed to create OpenShift web terminal workspace: %s %s", err.Error(), commandResult))
+			return
+		}
+
+		deploy, err := config.DevK8sClient.WaitDevWsStatus("web-terminal", config.WorkspaceNamespace, v1alpha1.WorkspaceStatusRunning)
+		if !deploy {
+			ginkgo.Fail(fmt.Sprintf("OpenShift Web terminal workspace didn't start properly. Error: %s", err))
+		}
+	})
+
+	var podName string
+	ginkgo.It("Check that pod creator can execute a command in the container", func() {
+		podSelector := "controller.devfile.io/workspace_name=web-terminal"
+		podName = config.AdminK8sClient.GetPodNameFromUserNameSpaceByLabel(podSelector, config.WorkspaceNamespace)
+		resultOfExecCommand, err := config.DevK8sClient.ExecCommandInContainer(podName, config.WorkspaceNamespace, "echo hello dev")
+		if err != nil {
+			ginkgo.Fail(fmt.Sprintf("Cannot execute command in the devworkspace container. Error: %s", err))
```

Comment:
It does not seem to tell a lot. Because of a bug I already fixed I've got

>   Cannot execute command in the devworkspace container. Error: exit status 1


So, it makes sense to include the full command output as well. I'll do it

Reply:
Done